catatonit.c

```diff
@@ -136,6 +139,81 @@ int kernel_signals[] = {SIGFPE, SIGILL, SIGSEGV, SIGBUS, SIGABRT, SIGTRAP, SIGSY
 
 #define ARRAY_LEN(arr) (sizeof(arr) / sizeof(*arr))
 
+static int syscall_close_range(unsigned int fd, unsigned int max_fd, unsigned int flags)
```

Comment:
I'd have preferred this be called `close_range` but it doesn't really matter.

Reply:
I've used the `syscall_` prefix to avoid conflicts with glibc `close_range`.

I guess I can add a check to `configure.ac` and define `close_range` only when it is not already present.

Reply:
Ah if I knew you were planning on doing that, I would've said it's okay to keep it as-is but thanks for doing that. :smile: